website/docs/d/api_management_api.html.markdown

```diff
@@ -59,6 +59,8 @@ output "api_management_api_id" {
 
 * `subscription_required` - Should this API require a subscription key?
 
+* `type` - The type of the API: "http", "graphql", "soap", "websocket"
```

Comment:
should this be `api_type` to make sure its exceptionally clear what "type" we are referring to?